examples/baseball.py

```diff
@@ -253,22 +272,38 @@ def main(args):
         .run(partially_pooled, at_bats, hits)
     logging.info("\nModel: Partially Pooled")
     logging.info("=======================")
+    logging.info("\nphi:")
+    logging.info(summary(posterior_partially_pooled, sites=["phi"],
+                         player_names=player_names)["phi"])
+    posterior_predictive = TracePredictive(partially_pooled,
+                                           posterior_partially_pooled,
+                                           num_samples=args.num_samples)
+    sample_posterior_predictive(posterior_predictive, baseball_dataset)
+    evaluate_log_predictive_density(partially_pooled, posterior_partially_pooled, baseball_dataset)
+
+    # (4) Partially Pooled with Logit Model
+    posterior_partially_pooled_with_logit = MCMC(nuts_kernel, num_samples=args.num_samples,
+                                                 warmup_steps=args.warmup_steps) \
+        .run(partially_pooled_with_logit, at_bats, hits)
+    logging.info("\nModel: Partially Pooled with Logit")
+    logging.info("==================================")
     logging.info("\nSigmoid(alpha):")
-    logging.info(summary(posterior_partially_pooled,
+    logging.info(summary(posterior_partially_pooled_with_logit,
                          sites=["alpha"],
                          player_names=player_names,
                          transforms={"alpha": lambda x: 1. / (1 + np.exp(-x))})["alpha"])
-    posterior_predictive = TracePredictive(partially_pooled,
-                                           posterior_partially_pooled,
+    posterior_predictive = TracePredictive(partially_pooled_with_logit,
+                                           posterior_partially_pooled_with_logit,
                                            num_samples=args.num_samples)
     sample_posterior_predictive(posterior_predictive, baseball_dataset)
-    evaluate_log_predictive_density(partially_pooled, posterior_partially_pooled, baseball_dataset)
+    evaluate_log_predictive_density(partially_pooled_with_logit,
+                                    posterior_partially_pooled_with_logit, baseball_dataset)
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Baseball batting average using HMC")
-    parser.add_argument("-n", "--num-samples", nargs="?", default=1200, type=int)
-    parser.add_argument("--warmup-steps", nargs='?', default=300, type=int)
+    parser.add_argument("-n", "--num-samples", nargs="?", default=200, type=int)
```

Comment:
I am guessing this was changed for testing? Should we revert the original defaults back.